### PR TITLE
Assessment 2: Classes, Objects, React, Jest

### PR DESCRIPTION
**Pull request by vanmeciv**
# Submitting Unit 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Coding Challenges
- [X] Coding Concepts
- [X] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -10,43 +10,143 @@
...
+
+const isDivisibleByThree = (number) => {
+    if (number % 3 === 0){
+        return `${number} is divisible by three`
+    } else if (number % 3 === 0) {
+        return `${number} is not divisible by three`
+    }
+}
+
+// console.log(isDivisibleByThree)
```
Comment (sjproctor) [inline code-challenges.test.js:51]: Remember that once you add Jest, there isn't a need for line 51

File: code-challenges.test.js [hunk 2]
```diff
@@ -10,43 +10,143 @@
...
+
+
+describe('isDivisibleByThree', () => {
+    it('-7 is not divisible by three', () => {
+      const object3 = { number: -7 }
+      const result3 = isDivisibleByThree(object3.number)
+      expect(result3)
+    })
+})
+// Output --> -7 is not divisible by three
```
Comment (sjproctor) [inline code-challenges.test.js:98]: All three expect statements can be handled by a single describe/it statement.

File: code-challenges.test.js [hunk 3]
```diff
@@ -10,43 +10,143 @@
...
+}
+
+// console.log(isDivisibleByThree)
+
+
+// --------------------2) Create a test with expect statements for each of the variables provided.
+// Psuedo: 
+// 1. describe("object1",() =>
+// 2. it("a function that takes an object as an argument and decides if the number inside it is evenly divisible by three or not.", () =>
+// 3. { expect (object1(15) ).toEqual(TRUE)}
```
Comment (sjproctor) [inline code-challenges.test.js:58]: The purpose of pseudo code is to plan your process in regular words not code

**Review by sjproctor — APPROVED** [review]
I can see you are exploring the concepts but the outcomes are still not there just yet. I would encourage you to focus on your pseudo coding in regular words so the problem gets clearly defined. Then you can use your tests to help you get to the right output. 
This assessment shows a novice level of skill level.

File: code-challenges.test.js (hunk 1 above)
Comment (vanmeciv) [reply code-challenges.test.js:51]: Thank you for reminding me I'm a novice. I thought agreeing with the LEARN policies made it clear I'm a student. i had the impression these were supposed to be novice concepts.

File: code-challenges.test.js (hunk 3 above)
Comment (vanmeciv) [reply code-challenges.test.js:58]: Pseudocode is a detailed yet readable description of what a computer program or algorithm should do (according to google), but I understand the interest in wanting to understand someone else's notes. I'd be happy to talk you through my thought process on the work.

**Pull request merged**